anaconda.py

```diff
@@ -80,15 +80,13 @@ def exitHandler(rebootData, storage):
     if anaconda.payload:
         anaconda.payload.unsetup()
 
+    # Unmount the filesystems.
     if not conf.target.is_hardware:
         anaconda.storage.umount_filesystems(swapoff=False)
-        devicetree = anaconda.storage.devicetree
-        devicetree.teardown_all()
```

Comment:
Are you sure this will work correctly? It looks like a behavior change.

Reply:
The method `teardown_all` is called in `teardown_disk_images`, so it shouldn't change image installations. And the device tree is not populated for dir installations, so it shouldn't change them either.

Reply:
OK, thanks for explanation.